md_api/src/main/java/com/wipro/ats/bdre/md/api/RegisterFile.java

```diff
@@ -47,16 +44,7 @@ public RegisterFile() {
             {"cTS", "creation-timestamp", "creation timestamp"},
             {"bid", "batch-id", "Batch id(use null for auto-generated batchid)"}
     };
-    /*private static RegisterFile registerFile;
-    public static RegisterFile getAutowiredRegisterFile(){
-        if(registerFile==null){
-            registerFile=new RegisterFile();
-            ApplicationContext appCtx = new ClassPathXmlApplicationContext("spring-dao.xml");
-            AutowireCapableBeanFactory acbFactory = appCtx.getAutowireCapableBeanFactory();
-            acbFactory.autowireBean(registerFile);
-        }
-        return registerFile;
-    }*/
+
     /**
      * This method runs RegisterFileProc proc in mysql and returns the input data back.
      *
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)